controllers/cloud.redhat.com/clowdapp_controller.go

```diff
@@ -243,15 +243,15 @@ func isOurs(meta metav1.Object, gvk schema.GroupVersionKind) bool {
 func ignoreStatusUpdatePredicate(logr logr.Logger, ctrlName string) predicate.Predicate {
 	return predicate.Funcs{
 		CreateFunc: func(e event.CreateEvent) bool {
-			gvk, _ := getKindFromObj(scheme, e.Object)
```

Comment:
Even though we've gotten rid of the unstructured approach, I kept this change in since getKindFromObj was defined twice in two files